### PR TITLE
windows form application solution

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:5]: !!!!!

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("WindowsFormsApplication1")]
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:8]: !!!!

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:11]: """""

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.Designer.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace WindowsFormsApplication1.Properties
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.Designer.cs:11]: name

**Review by ghfvs-tester — APPROVED** [review]

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx [hunk 5]
```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!--
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx:3]: la

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx [hunk 6]
```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx:5]: di

Comment (ghfvs-tester) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx:5]: da

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

File: WindowsFormsApplication1/WindowsFormsApplication1/Program.cs [hunk 7]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace WindowsFormsApplication1
+{
+    static class Program
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Program.cs:9]: 1

File: WindowsFormsApplication1/WindowsFormsApplication1/Program.cs [hunk 8]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace WindowsFormsApplication1
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Program.cs:7]: 2

File: WindowsFormsApplication1/WindowsFormsApplication1/Program.cs [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+{
+    static class Program
+    {
+        /// <summary>
+        /// The main entry point for the application.
+        /// </summary>
+        [STAThread]
+        static void Main()
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Program.cs:15]: another

File: WindowsFormsApplication1/WindowsFormsApplication1/Program.cs [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+        {
+            Application.EnableVisualStyles();
+            Application.SetCompatibleTextRenderingDefault(false);
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Program.cs:18]: review comment

**Review by ghfvs-tester — APPROVED** [review]

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx [hunk 11]
```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
+
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx]: ttttttt

Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx:7]: llllllloook

**Pull request merged**